### PR TITLE
XW-1214 add missing tracking to wiki navigation and unify existing one

### PR DESCRIPTION
**Pull request by bognix**
**Work in progress. Don't merge!**

## Links

* http://wikia-inc.atlassian.net/browse/XW-1214

## Description

Unify naming of events for side navigation on Mercury. Add missing events.

## Reviewers
@Wikia/x-wing

### CODE REVIEW

File: front/main/app/components/side-nav-global-navigation-root.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default Ember.Component.extend(
 
 		actions: {
 			goToLoginPage() {
-				this.goToLogin();
+				this.goToLogin('side-nav', 'open-login-signup');
```
Comment (Warkot) [inline front/main/app/components/side-nav-global-navigation-root.js]: `goToLogin` taking tracking parameters? Why not just add separate tracking function and track it here and not in the function itself?

**Comment by Warkot** [comment]
One comment, linter to fix and LGTM.

File: front/main/app/components/side-nav-global-navigation-root.js (hunk 1 above)
Comment (bognix) [inline front/main/app/components/side-nav-global-navigation-root.js]: there is tracking inside goToLogin

**Comment by vforge** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Pull request merged**